### ISSUE TITLE
password agreement looks disabled?

### ISSUE THREAD

**Issue by pro-wh** [oasisprotocol/oasis-wallet-ext#40]
![image](https://user-images.githubusercontent.com/38229038/129637336-3328c217-1cf7-46ce-ba81-08672dce8298.png)

it's light gray, which lightening the color usually means disabled, like with the "Next" button below. but it's enabled, and in fact you must check it before continuing

blocks #1

**Comment by sugarT365** [comment]
yes . you must need to click to agree , then you can go next . this just like `Use agreement `?
